corehq/apps/data_dictionary/tests/test_util.py

```diff
@@ -71,7 +71,15 @@ def test_already_existing_property(self, mock):
         self.assertEqual(CaseType.objects.filter(domain=self.domain).count(), 1)
         self.assertEqual(CaseProperty.objects.filter(case_type__domain=self.domain).count(), 1)
 
-        with self.assertNumQueries(2):
+        with self.assertNumQueries(3):
```

Comment:
It's really useful for code that is prone to n+1 queries. definitely caught myself accidentally adding sql queries that the tests caught